README.md

```diff
@@ -426,13 +426,14 @@ Join us on IRC at **#awesome-go** on freenode [web access](http://webchat.freeno
 * [go-fn](https://code.google.com/p/go-fn/) - Mathematical functions written in Go language, that are not covered by math pkg
 * [gofrac](https://github.com/anschelsc/gofrac) - A (goinstallable) fractions library for go with support for basic arithmetic.
 * [go-gt](https://code.google.com/p/go-gt/) - Graph theory algorithms written in "Go" language
-* [go.matrix](https://github.com/skelterjohn/go.matrix) - linear algebra for go
+* [go.matrix](https://github.com/skelterjohn/go.matrix) - linear algebra for go (has been stalled)
```

Comment:
If/when this PR gets merged, I'm planning to make another PR that removes the stale/inactive repos, espcially since there's a new/active one that is said to be better (can anyone confirm?). I don't think we want to keep such repos according to the [quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard):

> Actively maintained (even if that just means acknowledging open issues when they arise)